vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -215,6 +215,12 @@ namespace Microsoft.ReactNative
     DOC_DEFAULT("8081")
     UInt16 SourceBundlePort { get; set; };
 
+    DOC_STRING(
+      "When using a @.UseFastRefresh, @.UseLiveReload, or @.UseWebDebugger this controls whether the bundle URL requests inline source maps."
```

Comment:
Just noticed that there's a stray "a" in this and other doc strings, can you fix as long as you're here? ("When using a UseFastRefresh, ..." --> "When using UseFastRefresh")